src/index.js

```diff
@@ -1475,6 +1477,17 @@ export default class ReactJkMusicPlayer extends PureComponent {
           this.audio.pause()
           return
         }
+
+        if (restartCurrentOnPrev && !isNext) {
```

Comment:
`handlePlay` is common function, used by `audio reload`, `audio error play next audio` and `audio deleted`, so, do not modify this public method, you should change `onPlayPrevAudio` function, only affects the prev audio 

https://github.com/lijinke666/react-music-player/blob/7f4e19ef60752f4f37718ea361068c33892e90cd/src/index.js#L1553-L1555

like this

```tsx
  onPlayPrevAudio = () => {
    if(this.props.restartCurrentOnPrev && this.audio.currentTime > 1) {
      this.audio.currentTime = 0
     return
   }

    this.audioPrevAndNextBasePlayHandle(false)
  }
```
do you think so :) 😆

After you modify, I will immediately merge this PR. If you can't write unit test, I will write it by myself, thank you for your contribution